lib/datadog/tracing/contrib/sneakers/integration.rb

```diff
@@ -13,21 +13,33 @@ module Sneakers
         class Integration
           include Contrib::Integration
 
-          MINIMUM_VERSION = Gem::Version.new('2.12.0')
+          MINIMUM_SNEAKERS_VERSION = Gem::Version.new('2.12.0')
+          # All versions are supported. Kicks first version is 3.0.0.
+          MINIMUM_KICKS_VERSION = Gem::Version.new('3.0.0')
 
           # @public_api Changing the integration name or integration options can cause breaking changes
           register_as :sneakers, auto_patch: true
-
+          register_as_alias :sneakers, :kicks
+
+          # Sneakers development continues in the Kicks gem.
+          # The **only** thing that has changed is the gem name,
+          # even the file naming and module namespacing are the same (require 'sneakers', `::Sneakers`).
+          #
+          # The last version of Sneakers is 2.12.0.
+          # The first version of Kicks is 3.0.0. We currently support all versions of Kicks.
+          #
+          # @see https://github.com/jondot/sneakers/commit/9780692624c666b6db8266d2d5710f709cb0f2e2
           def self.version
```

Comment:
>They kept everything the same, even the file paths (require 'sneakers') and module names (::Sneakers).

`kicks` even continue the semantic version to avoid conflict with the obsolete gem.

Correct me if I am wrong. Due to this, I assume the easiest path is to support `kicks` is to change `self.version` with the conditional logic of `kicks` from `Gem.loaded_specs`. Do we need `register_as_alias` to support `kicks`?